content/bd/0001-intro-dbms.md

```diff
@@ -228,33 +228,33 @@ representa um único aluno.
 
 | `ist_id`   | `student_name` | `birthday` |
 | ---------- | -------------- | ---------- |
-| ist1123456 | Diogo          | 2002-03-06 |
-| ist1123234 | Rafa           | 2002-08-05 |
-| ist1124453 | Tomás          | 2002-12-18 |
-| ist1123534 | Diogo          | 2001-09-16 |
-| ist1123532 | Diogo          | 2002-04-27 |
+| ist123456 | Diogo          | 2002-03-06 |
+| ist123234 | Rafa           | 2002-08-05 |
+| ist124453 | Tomás          | 2002-12-18 |
+| ist123534 | Diogo          | 2001-09-16 |
+| ist123532 | Diogo          | 2002-04-27 |
```

Comment:
```suggestion
| ist1123456 | Diogo          | 2002-03-06 |
| ist1123234 | Rafa           | 2002-08-05 |
| ist1124453 | Tomás          | 2002-12-18 |
| ist1123534 | Diogo          | 2001-09-16 |
| ist1123532 | Diogo          | 2002-04-27 |
```
It does seem easier to read at first glance without the leading `1`, but student IST IDs always start with a `1` and the `123...` pattern is instinctively understood as a placeholder, which would make this informally-expressed regular expression technically incorrect.

This is definitely not that important though.